.github/ISSUE_TEMPLATE/task.yml

```diff
@@ -0,0 +1,50 @@
+name: Task
+description: Describe a generic activity we should carry out.
+labels: ["kind/task"]
+body:
+  - type: dropdown
+    id: Component
+    attributes:
+      label: Component
+      description: In which component did the issue occur?
+      options:
+        - Not sure*
+        - Operate*
+        - Optimize*
+        - Tasklist*
+        - Identity*
+        - Zeebe*
+        - C8Run*
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: what
+    attributes:
+      label: What should we do?
```

Comment:
Let's change this to "Description" to be more consistent with the style in the other templates.